csrc/cpu/spspmm_cpu.cpp

```diff
@@ -77,7 +77,7 @@ spspmm_cpu(torch::Tensor rowptrA, torch::Tensor colA,
             if (HAS_VALUE)
               tmp_vals[cB] += valA_data[eA] * valB_data[eB];
             else
-              tmp_vals[cB]++;
+              tmp_vals[cB] += 1;
```

Comment:
What's the reason for this change?

Reply:
The runtime error shows `++` not support BFloat16 implementation.